common/scripts/bump-beta-version.js

```diff
@@ -0,0 +1,23 @@
+const path = require('path');
+const { updateAllVersions, findAllPackageJSON, getAllNames, updateAllDepVersions } = require('./package-utils');
+
+const PACKAGES_DIR = path.join(__dirname, '..', '..', 'packages');
+
+const main = () => {
+  const packagePaths = findAllPackageJSON(PACKAGES_DIR);
+  const depNames = getAllNames(packagePaths);
+  updateAllVersions(bumpBetaVersion);
+  // Need to update all internal project dependencies using the same rule
+  updateAllDepVersions(bumpBetaVersion, depNames);
+}
+
+const bumpBetaVersion = (currentVersion) => {
```

Comment:
Instead of writing our own logic, does `beachball canary` cmd work? (what we use for creating alpha packages)

Reply:
That is a good point! Will beachball collect changelog and generate changelog files when using canary? Or it only updates all the versions in package.json

Reply:
I don't think so, I think it only bumps packages

Reply:
I tried canary method and looks like it doesn't follow our current bumping logic, since changing our bumping logic is another topic, I will keep this until canary supports this or we decide to change our bumping logic